bitnami/mariadb-galera/values.yaml

```diff
@@ -158,11 +158,14 @@ ldap:
   ## Enable LDAP support
   ##
   enabled: false
-  # uri:
-  # base:
-  # binddn:
-  # bindpw:
-  # bslookup:
+  uri:
```

Comment:
IMHO we should not leave these values un-commented in the non-production `values.yaml` file.